packages/plugin-core/src/_extension.ts

```diff
@@ -425,6 +429,21 @@ export async function _activate(
     toggleViews(false);
   }
 
+  if (extensionInstallStatus === InstallStatus.INITIAL_INSTALL) {
+    const VimInstalled = VSCodeUtils.isExtensionInstalled("vscodevim.vim");
+    if (VimInstalled) {
+      AnalyticsUtils.track(ExtensionEvents.VimExtensionInstalledInitial);
+      const [keybindingConfigPath, resolvedKeybindings] = getVimExtensionResolvedKeybinding();
```

Comment:
prefer object returns vs tuples. see [[prefer returning objects instead of tuples|dendron.dev.style#prefer-returning-objects-instead-of-tuples]]